### DISCUSSION TITLE
Is there an existing/recommended solution for handling password reset?

### DISCUSSION THREAD

**Discussion by andersr** [sergiodxa/remix-auth#241] [Q&A]
I am currently in the process adding remix-auth, including the FormStrategy, with email and password. I'd also like to support for password reset for that strategy.  I did a search and reviewed the docs and discussion forum looking for something pre-existing but did not find anything.  While I could roll my own, I wanted to first check to see if there is a pre-existing solution that anyone would recommend?  Thanks in advance for any tips or recommendations!

**Comment by sergiodxa** [comment]
I recommend you to check how @kentcdodds added password reset on [EpicStack](https://github.com/epicweb-dev/epic-stack).

**Comment by andersr** [comment]
@sergiodxa thanks for the tip! I'll take a look.
